R/aux_data_utils.R

```diff
@@ -624,39 +624,11 @@ gen_ili_data <- function(default_day_of_week = 1) {
     as_epi_archive(compactify = TRUE)
 }
 
-#' Process Raw NHSN Data
-#'
-#' Turns the raw NHSN data into a tidy format with the following columns:
-#' - geo_value: the jurisdiction of the data
-#' - disease: the disease of the data
-#' - time_value: the date of the data
-#' - version: the version of the data
-#' - value: the value of the data
-#'
-process_nhsn_data <- function(raw_nhsn_data) {
```

Comment:
removing this will break the `new_data.Rmd` report, along with the `forecast_dashboard`

Reply:
ah i need to update those